raco/algebra.py

```diff
@@ -799,6 +842,12 @@ def __eq__(self, other):
     def num_tuples(self):
         return self.input.num_tuples()
 
+    def partitioning(self):
```

Comment:
I don't think this can work for the same reason I stated under `IdenticalSchemeBinaryOperator`.